tests/.eslintrc.js

```diff
@@ -1,9 +1,9 @@
 /*eslint-env node*/
```

Comment:
I think this file should be deleted, as the root .eslintrc.js handles configuring the test file linting now

Reply:
Right, I just deleted it